src/foundry/common/utils/color.mjs.d.ts

```diff
@@ -0,0 +1,152 @@
+   */
+  get r(): number;
+
+  /**
+   * The numeric value of the green channel between [0, 1].
+   */
+  get g(): number;
+
+  /**
+   * The numeric value of the blue channel between [0, 1].
+   */
+  get b(): number;
+
+  /**
+   * The maximum value of all channels.
+   */
+  get maximum(): number;
+
+  /**
+   * The minimum value of all channels.
+   */
+  get minimum(): number;
+
+  /**
+   * Get the value of this color in little endian format.
+   */
+  get littleEndian(): number;
+
+  /**
+   * The color represented as an HSV array.
+   * Conversion formula adapted from http://en.wikipedia.org/wiki/HSV_color_space.
+   * Assumes r, g, and b are contained in the set [0, 1] and returns h, s, and v in the set [0, 1].
+   */
+  get hsv(): HSVColorVector;
+
+  /** @override */
+  toString(): string;
+
+  /**
+   * Test whether this color equals some other color
+   * @param other - Some other color or hex number
+   * @returns Are the colors equal?
+   */
+  equals(other: Color | number): string;
+
+  /**
+   * Get a CSS-compatible RGBA color string.
+   * @param alpha - The desired alpha in the range [0, 1]
+   * @returns A CSS-compatible RGBA string
+   */
+  toRGBA(alpha: number): string;
+
+  /**
+   * Mix this Color with some other Color using a provided interpolation weight.
+   * @param other - Some other Color to mix with
+   * @param weight - The mixing weight placed on this color where weight is placed on the other color
+   * @returns The resulting mixed Color
+   */
+  mix(other: Color, weight: number): Color;
+
+  /**
+   * Multiply this Color by another Color or a static scalar.
+   * @param other - Some other Color or a static scalar.
+   * @returns The resulting Color.
+   */
+  multiply(other: Color | number): Color;
+
+  /**
+   * Add this Color by another Color or a static scalar.
+   * @param other - Some other Color or a static scalar.
+   * @returns The resulting Color.
+   */
+  add(other: Color | number): Color;
+
+  /**
+   * Subtract this Color by another Color or a static scalar.
+   * @param other - Some other Color or a static scalar.
+   * @returns The resulting Color.
+   */
+  subtract(other: Color | number): Color;
+
+  /**
+   * Max this color by another Color or a static scalar.
+   * @param other - Some other Color or a static scalar.
+   * @returns The resulting Color.
+   */
+  maximize(other: Color | number): Color;
+
+  /**
+   * Min this color by another Color or a static scalar.
+   * @param other - Some other Color or a static scalar.
+   * @returns The resulting Color.
+   */
+  minimize(other: Color | number): Color;
+
+  /**
+   * Iterating over a Color is equivalent to iterating over its [r,g,b] color channels.
+   */
+  [Symbol.iterator](): Generator<number>;
+
+  /**
+   * Create a Color instance from an RGB array.
+   * @param color - A color input
+   * @returns The hex color instance or NaN
+   */
+  static from(color: null | string | number | RGBColorVector | Color): Color | typeof NaN;
```

Comment:
`typeof NaN` is just `number`. Unfortunately, we can't encode `NaN` in a type...

also, `Color` is missing from the param documentation in foundry, we should file an issue for that.

Reply:
I have an on going list of issues for documentation fixes to not spam them with like 10 separate ones.